src/components/coin-row/TransactionCoinRow.js

```diff
@@ -73,14 +75,53 @@ const TopRow = ({ balance, pending, status, title }) => (
 );
 
 export default function TransactionCoinRow({ item, ...props }) {
-  const { contact, hash } = item;
+  const { contact } = item;
   const { network } = useAccountSettings();
   const { navigate } = useNavigation();
 
   const onPressTransaction = useCallback(async () => {
-    const { from, to, status } = item;
-    const isPurchasing = status === TransactionStatusTypes.purchasing;
-    const isSent = status === TransactionStatusTypes.sent;
+    const { hash, from, minedAt, pending, to, status, type } = item;
+
+    const calculateTimestampOfToday = () => {
```

Comment:
we have all this functions at `src/helpers/transactions.js` so you might wanna export it there and import it here